### ISSUE TITLE
Impossible (softlocked) rando?

### ISSUE THREAD

**Issue by BrokenDays** [tewtal/itemrandomizerweb#62]
Hello everyone, I'm new to GitHub so sorry if I'm posting on the wrong section or anything.

As the title implies, I think my first hard rando I made isn't able to be completed and maybe I'm softlocked, all doors and places I need to go to progress requires Power Bombs or the Wave Beam, however I only have Missiles, Super Missiles, Speed Booster, Morph Ball, Morph Bombs and a ton of E-Tanks, and yes I got the "secret" items, such as the hidden E-Tank on the ceiling on the second missile room after getting the Morph Ball on the vanilla game, secret paths in general, and so on.
I'm very very familiar with the game and where to find the items, but I'm not a master on the game, so maybe I missed something.

I don't know if I missed one hidden item or room or something like that to give me a Power Bomb or the Wave Beam. I will be posting the seed here and my save file, so you don't need to play the whole game to get where I am at the moment.

Seed: HX8053115 (Hard) 
[The .zip files only includes the save files and save states, not the ROM itselft, for obvious reasons]

(Also it has various save states from ZSnes, the first one is the most recent and the furthest one, but the saved game is also the furthest I've got)

[Super Metroid Rando.zip](https://github.com/tewtal/itemrandomizerweb/files/4111251/Super.Metroid.Rando.zip)

If anyone plays it and is able to progress, find the Power Bomb and/or Wave Beam, etc or confirms that is a softlocked seed, please let me and everyone else knows.

Thanks in advance!

**Comment by moriwizz** [comment]
You missed the Power Bombs in Norfair. With the amount of health you have you can get this item and still make it out if you're good. https://i.imgur.com/LqHoB2D.png

**Comment by moriwizz** [comment]
Although, later on, it seems like Gravity Suit is at Spring Ball. As far as I know you can't get it without extremely difficult glitching.

**Comment by BrokenDays** [comment]
Yeah I found right now, forgot about that one, found it when I just gave myself 5 Power Bombs by using game genie codes and went item hunting and remembered that hidden item going back to Brinstar :p

Really? I didn't know that "Hard" means this hard xD
I will try it without cheats or anything like that, but if I can't I maybe just use a code to get it like I did with the Power Bombs so I can complete the rando, after all this first rando was just to test myself with it before recording and all that.

But thanks a lot for the reply and the heads up on the Gravity Suit, gonna see if I can do it :)

**Comment by InsaneFirebat** [comment]
You should be able to complete the game suitless. There are several tricks to help you navigate Maridia without Gravity. See Reverse Boss Order speedruns and wiki page for reference.

https://wiki.supermetroid.run/RBO
